CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 - Changed `ex.Scene.children` to be `ex.Scene.actors`, and updated unit test specs to reflect this change ((#796)[https://github.com/excaliburjs/Excalibur/issues/796])
 - Added backgroundColor to engine options as part of Engine constructor ((#846)[https://github.com/excaliburjs/Excalibur/issues/846])
+- Added options for hsl,hex representation in Color.toString() method with rgb as default if none provided to maintain backward compatability((#852)[https://github.com/excaliburjs/Excalibur/issues/852])
```

Comment:
Since this adds new output formats, it might be a better fit for Added:

- Added new `hsl` and `hex` format options in `Color.toString(format)` using `rgb` as the default to maintain backwards compatibility ([#852](https://github.com/excaliburjs/Excalibur/issues/852))

I think the other issue links above are using wrong Markdown syntax too, it's `[text](link)` instead of the other way.